src/main/java/io/jenkins/plugins/models/JiraIssue.java

```diff
@@ -0,0 +1,47 @@
+package io.jenkins.plugins.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class JiraIssue {
+    @JsonProperty("key")
```

Comment:
you should only need json property to rename keys

Reply:
Hrm, interesting.

The other models do it, and I figured consistency and clarity was a good plan.

These are used for output not input, since jira's json is hella nested messy